tasks/configure.yml

```diff
@@ -41,7 +41,7 @@
 
 - name: PostgreSQL | Reset the cluster - create a new one (with specified encoding and locale) | Debian
   shell: >
-    pg_createcluster --start --locale {{ postgresql_locale }}
+    pg_createcluster --locale {{ postgresql_locale }}
```

Comment:
I totally agree with this behaviour but this should be made optional and the default not change.